### PR TITLE
fix(cloudwatch-logs): Add missing variables to allow compare services to put logs

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
The compare MMDL & the Appian compare services were getting an error trying to put watch logs. They were missing namespaces which caused the cloudwatch logs to throw an error. The Appian compare was also missing an KMS developer role which prevented the compare-appian service from posting logs.

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/240?modal=detail&selectedIssue=OY2-23005

### CODE REVIEW

File: src/services/compare-appian/serverless.yml [hunk 1]
```diff
@@ -198,7 +209,7 @@ resources:
 stepFunctions:
   stateMachines:
     compareAlertFunc:
-      name: ${self:service}-${sls:stage}-compare-appian-alerting
+      name: ${self:service}-${sls:stage}-compare-alerting-appian
```
Comment (benjaminpaige) [inline src/services/compare-appian/serverless.yml:212]: can you do me a favor and change the compareAlertFunc name in compare-mmdl to match this pattern?

File: src/services/compare-mmdl/serverless.yml [hunk 2]
```diff
@@ -143,12 +143,14 @@ functions:
       region: ${self:provider.region}
       statusTableName: ${param:statusTableName}
       errorsTopicArn: ${param:errorsTopicArn}
+      namespace: ${self:service}-${sls:stage}
```
Comment (benjaminpaige) [inline src/services/compare-mmdl/serverless.yml:146]: thanks for catching that 👍

File: src/services/connector/serverless.yml [hunk 3]
```diff
@@ -566,6 +592,19 @@ resources:
...
+        AlarmActions:
+          - ${param:alarmsTopicArn}
+        ComparisonOperator: GreaterThanOrEqualToThreshold
+        DatapointsToAlarm: 1
+        EvaluationPeriods: 5
+        MetricName: ${param:appianTableName}_dynamo_updates
+        Namespace: ${self:service}-${sls:stage}
+        Period: 60
+        Statistic: Sum
+        Threshold: 1
```
Comment (benjaminpaige) [inline src/services/connector/serverless.yml:607]: 🔥 alarms

**Review by benjaminpaige — APPROVED** [review]
looks great yo! Just the one small ask.

File: src/services/compare-appian/serverless.yml (hunk 1 above)
Comment (jdinh8124) [reply src/services/compare-appian/serverless.yml:212]: Done!

**Pull request merged**